share/jupyterhub/templates/token.html

```diff
@@ -3,6 +3,7 @@
 {% block main %}
 
 <div class="container">
+  <h1 class="sr-only">Token page</h1>
```

Comment:
Rather than `Token Page`, let's do something more descriptive like `Manage JupyterHub Tokens`.

Reply:
Okay. I will rename the h1 tags now